packages/x-charts/src/ChartsTooltip/ChartsItemTooltipContent.tsx

```diff
@@ -37,13 +37,17 @@ function ChartsItemTooltipContent(props: ChartsItemTooltipContentProps) {
       <ChartsTooltipTable className={classes.table}>
         <tbody>
           <ChartsTooltipRow className={classes.row}>
-            <ChartsTooltipCell className={clsx(classes.markCell, classes.cell)}>
+            <ChartsTooltipCell
+              className={clsx(classes.markCell, classes.cell)}
+              component="td"
+              aria-hidden="true"
+            >
```

Comment:
We could skip this entire column, which should allow us a consistent margin style across content.

This is `aria-hidden` anyways, useless for tabular data. 

If we remove it altogether and add the contents to the `labelCell`

```tsx
            <ChartsTooltipCell className={clsx(classes.labelCell, classes.cell)} component="th">
              <ChartsLabelMark type={markType} color={color} className={classes.mark} />{label}
            </ChartsTooltipCell>
```

Then we would just need to update some margins and styles and everything should look as expected

Reply:
I had to add a wrapper for the tooltip mark to keep series label aligned when shapes are different, because the square has `width: 13` but line has `width: 16`

So to simplify I dedicated 20px + theme.spacing(1.5) for the mark